package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "@viamrobotics/sdk",
-  "version": "0.29.0",
+  "version": "0.29.1",
```

Comment:
Ok to minor bump?

Reply:
We don't update the version manually but rather through a workflow when it's time to release, so I would change this back please

Reply:
Switching from CJS to ESM should be considered a breaking change, since it will prevent existing CJS consumers from importing the package. Since this package is `0.*` (`<1.0.0`), a bump to `0.30.0` satisfies semver requirements for communicating a break

Reply:
SGTM. Done in ad6e215

Reply:
Just realized I misspoke a little here: we've already got a CJS package export defined, so probably this won't break any consumers as written.

The being said, I've seen other libraries consider touching type and exports to be breaking changes, so I think erring on the side of caution here and assuming potential downstream breakage is a good idea